### ISSUE TITLE
New Resource: BucketCORSPolicy

### ISSUE THREAD

**Issue by JacksonMaxfield** [pulumi/pulumi-google-native#202]
## Affected area

Adding a new resource for setting / updating a buckets CORS policy.

https://cloud.google.com/storage/docs/configuring-cors#gsutil

If this should be placed on the gcloud pulumi repo instead of this native repo all good. Just would love to see this resource exist.

**Comment by mikhailshilkov** [comment]
It looks like one can set CORS on the `Bucket` resource: https://github.com/pulumi/pulumi-google-native/blob/master/sdk/python/pulumi_google_native/storage/v1/bucket.py#L20

Does this not work for you?

**Comment by JacksonMaxfield** [comment]
Nope. My bucket is automatically created by Google app engine. https://www.pulumi.com/docs/reference/pkg/gcp/appengine/application/#outputs

**Comment by mikhailshilkov** [comment]
I see... So you want to patch an existing resource that is managed elsewhere?

**Comment by JacksonMaxfield** [comment]
Hmmm I hadn't thought of it that way but fair point.

I originally thought "I would like to set CORS on a bucket." Figuring all buckets under the hood are the same.
